<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyBP</class>
 <widget class="QDialog" name="ModifyBP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of days after the due date :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBonusPenaltyType">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="intBonusPenaltyId">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bonus Penalty Type ID :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="intAmount">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Percentage of Bonus Penalty :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Type: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bonus Penalty ID: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_updateBonusPenalty">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Bonus Penalty </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="intHour">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_BPTypeID">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BonusPenaltyTypeID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
